dts/bindings/rtc/nxp,kinetis-pit.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2020 NXP Semiconductor INC.
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP MCUX Periodic Interrupt Timer (PIT)
+
+compatible: "nxp,kinetis-pit"
+
+include: [rtc.yaml]
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    label:
+      required: true
```

Comment:
These properties are required in `rtc.yaml`, so it's redundant to put them here too. You can remove these lines.

Reply:
ok